roles/kubernetes/master/defaults/main.yml

```diff
@@ -20,6 +20,11 @@ kube_apiserver_storage_backend: etcd3
 # By default, force back to etcd2. Set to true to force etcd3 (experimental!)
 force_etcd3: false
 
+# Associated interfaces must be reachable by the rest of the cluster, and by
+# CLI/web clients.
+kube_controller_manager_bind_address: 0.0.0.0
+kube_scheduler_bind_address: 0.0.0.0
```

Comment:
Since the default prior to this PR was `localhost`, I would use that here, especially since this might have the undesirable effect for other kubespray users of causing their scheduler and controller managers to listen on all interfaces.